src/js/components/footer.js

```diff
@@ -0,0 +1,41 @@
+import { newText, newMultimedia, newLink, newContainer } from './helpers';
+
+const createLinks = (dataLinks) =>{
+  const arrayLinks = dataLinks.map((link)=>{
+    const linksItems = newLink(link.href,link.text,['footer__data-links']);
+    const listLink = newContainer('LI',[linksItems],['footer__items-list']);
+    return listLink;
+  });
+  return arrayLinks;
+}
+
+const createIcons = (dataIcons) => {
+  const arrayIcons = dataIcons.map((icons)=>{
+    const iconsImages = newMultimedia('IMG',icons.icon.src,icons.icon.alt,['footer__icons']);
+    const iconsLinks = newLink(icons.href,'',['footer__icons-links']);
+    iconsLinks.target = '_blank';
+    iconsLinks.appendChild(iconsImages);
+    return iconsLinks
+  });
+  return arrayIcons;
+}
+
+
+const footerComp = (data) => {
+
+  const logoFooter = newMultimedia('IMG', data.logo.image.src, data.logo.image.alt,['footer__logo']);
```

Comment:
change Capitalize code IMG to lowercase.

Reply:
I already  changed everything I had in capital letters to lowercase, in order to  standard all code